.github/workflows/dapr_cli.yaml

```diff
@@ -35,6 +35,9 @@ jobs:
       GOARCH: ${{ matrix.target_arch }}
       GOPROXY: https://proxy.golang.org
       ARCHIVE_OUTDIR: dist/archives
+      DaprEXEDirectory: dist/windows_amd64/release
+      DaprLicenseDirectory: .github/wix
+      WIX_BIN_PATH: 'C:/Program Files (x86)/WiX Toolset v3.11/bin'
```

Comment:
Wix tool set is already installed in the vm?

Reply:
Yes it's there by default in windows runner: https://github.com/actions/runner-images/blob/main/images/win/Windows2022-Readme.md